content/en/blog/_posts/2018-12-05-kubeadm-ga-release.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: blog
+title: Production-Ready Kubernetes Cluster Creation with kubeadm
+date: 2018-12-05
+---
+
+**Authors**: Lucas Käldström (CNCF Ambassador) and Luc Perkins (CNCF Developer Advocate)
+
+[kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/) is a tool that enables Kubernetes administrators to quickly and easily bootstrap minimum viable clusters that are fully compliant with [Certified Kubernetes](https://github.com/cncf/k8s-conformance/blob/master/terms-conditions/Certified_Kubernetes_Terms.md) guidelines. It's been under active development by [SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle) since 2016 and we're excited to announce that it has now graduated from beta to stable and generally available (GA)!
+
+This GA release of kubeadm is an important event in the progression of the Kubernetes ecosystem, bringing stability to an area where stability is paramount.
+
+The goal of kubeadm is to provide a foundational implementation for bootstrapping common cluster setup tasks and for cluster administration. kubeadm ships with best-practice defaults but can also be customized to support other ecosystem requirements or vendor-specific approaches. kubeadm is designed to be easy to integrate into larger deployment systems and tools.
```

Comment:
"for bootstrapping common cluster setup tasks and for cluster administration." --> what about simplifying in "for cluster setup and for cluster administration."